src/containers/single-event/SingleEvent.tsx

```diff
@@ -0,0 +1,92 @@
+import { Spin } from 'antd';
```

Comment:
lowerCamelCase directories and rename the container to `index.tsx` to use shortened imports in App